src/main/webapp/script.js

```diff
@@ -0,0 +1,17 @@
+//implementing map
+var map;
+function initMap() {
+    const map = new google.maps.Map(
+        document.getElementById('map'),
+        {center: {lat: 44.9740, lng: -93.2277}, zoom: 18}
+    );
+    
+    const trexMarker = new google.maps.Marker({
+        position: {lat: 44.9740, lng: -93.2277},
+        map: map, 
+        title: 'Afro Deli'
```

Comment:
Later on, this could be part of the data you fetch from the backend

Reply:
Will make sure to do that in the future!